projects/cds/src/cds-models/cds-strategy-request.model.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * A model class that is used when sending a request to the Context-Driven Services Merchandising Strategy service for returning
+ * products for a carousel. There are a number of things that can affect which products are returned, based on the users current
+ * context
+ */
+export interface StrategyRequest {
+  productId?: string;
+  category?: string;
+  facets?: string;
```

Comment:
Is this a comma separated list? Asking because it's in plural.

Reply:
It is something like that yes. I've yet to see what this is in Spartacus, but its a string separated by ':' I believe